drivers/ds323x/include/ds3234_params.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 SKF AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ds3234
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for DS3234 devices
+ *
+ * @author      Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ */
+
+#ifndef DS3234_PARAMS_H
+#define DS3234_PARAMS_H
+
+#include "board.h"
+#include "ds3234.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters for the DS3234 devices
+ * @{
+ */
+#ifndef DS3234_PARAM_SPI
+#define DS3234_PARAM_SPI                (SPI_DEV(0))
+#endif
+#ifndef DS3234_PARAM_CS
+#define DS3234_PARAM_CS                 (SPI_HWCS(0))
```

Comment:
I think `GPIO_PIN(0, 0)` would be the more common way here.

Reply:
Changed